source/app/menu.cpp

```diff
@@ -45,6 +45,8 @@ void showMainMenu() {
         WHBLogFreetypePrintf(L"%C Dump only DLC files of a game", OPTION(6));
         WHBLogFreetypePrintf(L"%C Dump only Save files of a game", OPTION(7));
         WHBLogFreetypePrintf(L"%C Dump whole MLC (everything stored on internal storage)", OPTION(8));
+        WHBLogFreetypePrint(L"");
+        WHBLogFreetypePrintf(L"%C Dump all SpotPass data", OPTION(9));
```

Comment:
Dumping one file isn't really "all" SpotPass data. I think this option should also just be called something like `Send SpotPass Data To Archival Project`/`Dump SpotPass Data`/`Contribute SpotPass Data`.